project/CodeCoverageSettings.scala

```diff
@@ -31,9 +31,10 @@ object CodeCoverageSettings {
     "testOnlyDoNotUseInAppConf.*"
```

Comment:
`".*\\$anon.*"` needs adding to this list. see relevent file [here](https://github.com/hmrc/self-assessment-bsas-api/pull/581/files#r2218620994) for example and explination